Library/Homebrew/formula.rb

```diff
@@ -775,6 +775,8 @@ def to_hash
           "used_options" => tab.used_options.as_flags,
           "built_as_bottle" => tab.built_bottle,
           "poured_from_bottle" => tab.poured_from_bottle
+          if ENV["HOMEBREW_CXX"]: "CXX_Compiler" => ENV["HOMEBREW_CXX"]
```

Comment:
This isn't valid Ruby syntax. You can't include conditionals like this in the middle of a hash literal, and Ruby's `if` statements are structured differently from Python ones.